internal/connector/test/integration/features/connector-agent-api.feature

```diff
@@ -299,7 +299,50 @@ Feature: connector agent API
           "version": "1.0",
           "namespace": "openshift-mcs-camelk-1.0",
           "status": "ready"
-        }]
+        }],
```

Comment:
at this stage, the test is mainly to ensure that the change to the OpenAPI has no side effect